python/kernmlops/kernmlops_benchmark/redis.py

```diff
@@ -123,6 +127,8 @@ def run(self) -> None:
                     "-p",
                     "redis.port=6379",
                     "-p",
+                    "redis.timeout=60000",
+                    "-p",
```

Comment:
Why are we adding a timeout here?

Reply:
Timeouts are now optional. Enabled for when PIN is enabled as Redis commands are much slower with the tool enabled and lead to benchmark timeouts without them.